### ISSUE TITLE
Sync eng/common from Azure/azure-sdk-tools

### ISSUE THREAD

**Issue by heaths** [Azure/azure-sdk-for-rust#1601]
Sync `eng/common` from [Azure/azure-sdk-tools](https://github.com/Azure/azure-sdk-tools).

**Comment by hallipr** [comment]
This is still a manual sync process until track 2 merges to main.  I'll keep the ticket open until we have the process automated

**Comment by heaths** [comment]
No. We're getting sync'd updates. I've even done a couple myself.
